starlette/templating.py

```diff
@@ -99,7 +99,7 @@ def __init__(
         if directory is not None:
             self.env = self._create_env(directory, **env_options)
         elif env is not None:
```

Comment:
Can we have this instead?
```suggestion
        elif env is not None:  # pragma: no branch
```